common/common.go

```diff
@@ -72,6 +72,7 @@ var (
 	ErrUnknownError           = errors.New("unknown error")
 	ErrGettingField           = errors.New("error getting field")
 	ErrSettingField           = errors.New("error setting field")
+	ErrParsingWSField         = errors.New("error parsing WS field")
```

Comment:
```suggestion
	ErrParsingWSField         = errors.New("error parsing websocket field")
```

This wasn't your doing, but I'd prefer that we don't use acronyms for error messages